docs/source/markdown/podman-attach.1.md

```diff
@@ -20,9 +20,7 @@ it in the **libpod.conf** file: see **libpod.conf(5)** for more information.
 ## OPTIONS
 **--detach-keys**=*sequence*
 
-Override the key sequence for detaching a container. Format is a single character `[a-Z]` or
-a comma separated sequence of `ctrl-<value>`, where `<value>` is one of:
-`a-z`, `@`, `^`, `[`, `\\`, `]`, `^` or `_`.
+Specify the key sequence for detaching a container. Format is a single character `[a-Z]` or `ctrl-<value>` where `<value>` is one of: `a-z`, `@`, `^`, `[`, `,` or `_`. Specifying "" will disable this feature. The default is *ctrl-p,ctrl-q*.
```

Comment:
It's not a single character, though - it's one or more characters.

Reply:
you say "`[a-Z]`" and then "`a-z`" is it all lower chars or are upper allowed too?

Reply:
That is how the Code works.  And this is how Docker was designed.
```
/bin/podman run -ti --detach-keys "Z" fedora  /bin/sh
sh-5.0# 
```
```
./bin/podman run -ti --detach-keys "ctrl-Z" fedora  /bin/sh
Error: invalid detach keys: Unknown character: 'ctrl-Z'
./bin/podman run -ti --detach-keys "ctrl-z" fedora  /bin/sh
sh-5.0# 
```

Reply:
OK, then I think you need to change "one of : `a-z`" to "one of: `a-Z`" or I'd prefer "one of: `a-z`, `A-Z`"